Resources/Core/Build/Scripts/splitAcceptanceTests.php

```diff
@@ -80,7 +80,7 @@ public function execute()
         // Find functional test files
         $testFiles = (new Finder())
             ->files()
-            ->in(__DIR__ . '/../../../../../../../typo3/sysext/core/Tests/Acceptance/Backend')
+            ->in([__DIR__ . '/../../../../../../../typo3/sysext/core/Tests/Acceptance/Backend', __DIR__ . '/../../../../../../../typo3/sysext/core/Tests/Acceptance/InstallTool'])
```

Comment:
I'm wondering why this file is at all in testing-framework and not in TYPO3 Core repo...
Acceptance tests for **backend** in **core** and now for **install tool** in **core**...
I think there's room for improvement in the future ;)